core/src/codegen/from_meta_impl.rs

```diff
@@ -90,16 +89,37 @@ impl<'a> ToTokens for FromMetaImpl<'a> {
                     }
                 };
 
+                let default_or_err = base.default.as_ref().map_or(
+                    quote!(::darling::export::Err(::darling::Error::too_few_items(1))),
+                    |default_expr| quote!(::darling::export::Ok(#default_expr)),
+                );
+
+                let word_or_err = variants
+                    .iter()
+                    .find_map(|variant| {
+                        if variant.word {
```

Comment:
There should be something that errors if two variants on the same enum both have `word` set; one of those is going to be ignored every time, and it's indicative of a dev error that darling should protect against.

Unfortunately, the check can't go here, as this method is expected to be infallible. It also can't go in the individual variants, as those don't have the visibility needed to do the check.

An alternative would be to hoist `word` to be settable on the enum itself, so you'd write `#[darling(word = Sit)]`; that has the advantage of being explicit that there can only be one variant which has this behavior, but it feels "wrong" somehow to me.

Reply:
I agree that hoisting to the enum feels "wrong" 🤔.

I think there should also be an error when a variant is annotated with both `skip` and `word`, does that make sense?

Reply:
I could actually see a use-case for using `skip` with `word` - it would let you make a variant that was only addressable in word form.